db/migrate/20210616073326_create_contributor_teams.rb

```diff
@@ -0,0 +1,24 @@
+class CreateContributorTeams < ActiveRecord::Migration[6.1]
+  def change
+    create_table :contributor_teams do |t|
+      t.belongs_to :track, null: true, foreign_key: true
+      t.string :name, null: false, index: { unique: true }
+      t.string :github_name, null: false, index: { unique: true }
+      t.column :type, :tinyint, null: false, default: 0      
+
+      t.timestamps
+    end
+
+    create_table :contributor_team_memberships do |t|
+      t.belongs_to :contributor_team, null: false, foreign_key: true
+      t.belongs_to :user, null: false, foreign_key: true
+      
+      t.column :visible, :boolean, null: false, default: true
+      t.column :seniority, :tinyint, null: false, default: 0
+
+      t.index [:contributor_team_id, :user_id], name: "index_contributor_team_memberships_on_team_id_and_user_id", unique: true
+
+      t.timestamps
+    end
```

Comment:
It's best to generally split seperate table creates into seperate migrations.

The reason for this is that if the first succeeds, but the second fails, you'll be in a state where you can't re-run the migration, and can't rollback easily either. 

If you want to still use redo/rollback, you can do `rails db:migrate:redo STEP=2`

Reply:
Yeah I got bitten by that. Will update.